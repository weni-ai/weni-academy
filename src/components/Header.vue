<template>
  <header class="unnnic-grid-span-12">
    <unnnicBreadcrumb
      :crumbs="[
        { name: 'One', path: 'One' },
        { name: 'Two', path: 'Two' },
        { name: 'Active', path: 'Active' },
      ]"
    />
  </header>
</template>

<script>
export default {};
</script>

<style scoped lang="scss">
  header {
    padding: 0 $unnnic-spacing-stack-md;
  }
</style>