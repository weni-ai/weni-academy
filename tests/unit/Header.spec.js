import { shallowMount } from '@vue/test-utils';
// import Vuex from 'vuex';
// import Router from 'vue-router';
import Header from '@/components/Header.vue';
// import i18n from '@/utils/plugins/i18n';

// const localVue = createLocalVue();

// localVue.use(Vuex);
// localVue.use(Router);

// const router = new Router();

describe('Header.vue', () => {
  // let getters;
  // let actions;
  // let store;
  let wrapper;

  beforeEach(() => {
    // getters = {
    //   currentOrg: () => {
    //     return {
    //       name: 'oi',
    //       uuid: '12',
    //       inteligence_organization: 'topp',
    //       authorization: 'true',
    //     };
    //   },
    // };
    // actions = {
    //   setCurrentProject: jest.fn(),
    //   createProject: jest.fn(),
    // };

    // store = new Vuex.Store({
    //   getters,
    //   actions,
    // });

    wrapper = shallowMount(Header, {
      // localVue,
      // i18n,
      // store,
      // router,
      propsData: {},
      stubs: {
        unnnicBreadcrumb: true
      },
    });
  });

  it('renders a snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
