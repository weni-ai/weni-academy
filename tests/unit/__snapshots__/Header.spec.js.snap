// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.vue renders a snapshot 1`] = `
<header class="unnnic-grid-span-12">
  <unnnicbreadcrumb-stub crumbs="[object Object],[object Object],[object Object]"></unnnicbreadcrumb-stub>
</header>
`;
